### DISCUSSION TITLE
抓包报错

### DISCUSSION THREAD

**Discussion by xianyulao** [gojue/ecapture#477] [Q&A]
手机型号1+ acepro 安卓12，内核版本5.10.101，ecapture版本0.7.3
运行./ecapture tls提示Your environment is like a container. We won't be able to detect the BTF configuration.百度了一下说是无法检测BTF配置,这是这么情况,777权限给了

**Comment by cfc4n** [comment]
提示不用管，能正常运行么？ 
如果不能，就换 nocore版本试试。

**Reply by xianyulao** [reply]
感谢大佬，一开始我以为不能用然后没管他，我忙完一圈回来发现它又抓到包了，不过有的app可以有的不行，抓到的也都是加密后的☺️

**Reply by cfc4n** [reply]
多看解说视频、介绍文章。博主很忙，可能没时间一一解答。

**Reply by BiteFoo** [reply]
这种应该是原始数据就是 经过加密了，只是再加密数据的基础上使用https传输。因此，如果需要重新解包，需要分析 apk 的加密数据方法，试试下面的这种方式
1、先把请求的 body 或者响应数据保存
2、寻找加密解密的地方应该就可以了

我实际使用的过程中也有会有抓包数据是被加密的情况，需要逆向分析来实现数据的解密

**Reply by xianyulao** [reply]
我一开始以为是我没配置好，后来发现塔那个报错的日志也能抓包，和小黄鸟差不多
